src/jit/assertionprop.cpp

```diff
@@ -367,22 +367,17 @@ void                Compiler::optAddCopies()
                     continue;
                 }
 
-                // This block will be the new candidate for the insert point
-                // for the new assignment
-                //
 #ifdef DEBUG
                 if  (verbose)
                     printf("new bestBlock\n");
 #endif
 
+                // This block will be the new candidate for the insert point
```

Comment:
Various comments were moved either below and #ifdef, inside the ifdef, or above something right above and ifdef so that clang-format will not modify the indentation of the comment.